cosmoz-bottom-bar.js

```diff
@@ -402,6 +403,14 @@ class CosmozBottomBar extends PolymerElement {
 			this.style.maxHeight = to;
 		});
 	}
+
+	_listenForButtonClicks() {
+		window.addEventListener('keydown', (event) => {
```

Comment:
We can't add a event listener and not clean it up at some point ....

Reply:
https://polymer-library.polymer-project.org/3.0/docs/devguide/events#listener-on-outside-elements